performance_regression_test.py

```diff
@@ -375,11 +367,24 @@ def test_mixed(self):
         stress_queue = self.run_stress_thread(stress_cmd=base_cmd_m, stress_num=2)
         results = self.get_stress_results(queue=stress_queue, stress_num=2)
 
-        try:
-            self.display_results(results, test_name='test_mixed')
-        except:
-            pass
+        self.display_results(results, test_name='test_mixed')
         self.generate_stats_json(results, [base_cmd_w, base_cmd_m])
 
+    def test_user_profile(self):
+        """
+        Run workload using user profile(-s)
+        """
+        for cs_profile in self.params.get('sc_user_profiles').split():
+            self.log.debug('Run stress test with user profile {}'.format(cs_profile))
+            profile_dst = os.path.join('/tmp', os.path.basename(cs_profile))
```

Comment:
We'll use the profiles from scylla-qa-internal repo. I cannot define the full path in the yaml file, so I'll add cloning of the repo and replace file names with fullpath names in jenkins script.
If you'll have some better idea, please advise.